src/main/java/picard/fingerprint/CheckFingerprint.java

```diff
@@ -51,14 +51,57 @@
 import java.util.List;
 
 /**
- * Attempts to check the sample identity of the sequence/genotype data in the provided file (SAM/BAM or VCF)
- * against a set of known genotypes in the supplied genotype file (in either GELI or VCF format).
+ * Check the sample identity of the sequence/genotype data in the provided file (SAM/BAM or VCF)
+ * against a set of known genotypes in the supplied genotype file (in VCF format).
  *
- * @author Tim Fennell
+ * <h4> Summary </h4>
+ * Computes a fingerprint from the supplied input file (SAM/BAM or VCF) file and
+ * compares it to the expected fingerprint genotypes provided. The key output is a LOD score
+ * which represents the relative likelihood of the sequence data originating from the same
+ * sample as the genotypes vs. from a random sample.  Two outputs are produced: (1) a summary
+ * metrics file that gives metrics of fingerprint match when comparing the input to a set of
+ * genotypes for the expected sample.  at the single sample level (if the input was a VCF) or at the read
+ * level (lane or index within a lane) (if the input was a SAM/BAM) and (2) a detail metrics file that
+ * contains an individual SNP/Haplotype comparison within a fingerprint comparison.  The metrics files
+ * fill the fields of the classes {@link FingerprintingSummaryMetrics} and {@link FingerprintingDetailMetrics}.
+ * The output files may be specified individually using the SUMMARY_OUTPUT and DETAIL_OUTPUT options.
+ * Alternatively the OUTPUT option may be used instead to give the base of the two output
+ * files, with the summary metrics having a file extension ".{@value CheckFingerprint#FINGERPRINT_SUMMARY_FILE_SUFFIX}",
+ * and the detail metrics having a file extension ".{@value CheckFingerprint#FINGERPRINT_DETAIL_FILE_SUFFIX}".
+ * <p>
+ * <h4>Example</h4>
+ * <h3>Comparing a bam against known genotypes:</h3>
+ * <pre>
+ *     java -jar picard.jar CheckFingerprint \\ <br />
```

Comment:
I think you need to remove the repeated slash and the ```<br />``` as these would add empty lines within the &lt;pre /&gt;